tailwind.config.js

```diff
@@ -315,8 +319,17 @@ module.exports = {
       redirect: '150',
     },
   },
-  variants: {},
-  plugins: [],
+  variants: {
+    extend: {
+      borderColor: ['focus-within', 'focus', 'hover'],
+      backgroundColor: ['focus-within'],
+    },
+  },
+  plugins: [
+    function ({ addVariant }) {
+      addVariant('child', '& > *')
```

Comment:
Where is this used?

Reply:
in PreviewFooter.tsx and SubmittedAutomationModal.tsx.
The reason is the same: it leaves those components untouched, but does some minor styling on them. Eg. tabs or the reputation badge components, which I didn't want to modify.

Reply:
and from Tailwind 3.1 you get this support out of the box without having to create a plugin for it.
https://tailwindcss.com/blog/tailwindcss-v3-1#arbitrary-values-but-for-variants